docs/2.9.0/docs/canton/usermanual/support_checklist.rst

```diff
@@ -0,0 +1,129 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates.
+..
+   Proprietary code. All rights reserved.
+
+.. _support_checklist:
+
+Support Checklist
+=================
+
+In order to successfully provide support, but also to ensure that you are able to troubleshoot the system
+efficiently on your own, we recommend that you follow the checklist below to ensure that you have access to
+all the necessary information and tools, and that you have established the necessary administrative access
+to your production system.
```

Comment:
```suggestion
So that we can successfully provide support, and to ensure that you can troubleshoot the system
efficiently on your own, we recommend that you follow the checklist below. This ensures that you have access to
all the necessary information and tools to resolve an issue and that you have established the necessary administrative access
to your production system.
```

Reply:
fixed